apps/studio/src/features/dashboard/SiteList.tsx

```diff
@@ -115,13 +116,47 @@ const SuspendableSiteList = (): JSX.Element => {
       <Flex flexDirection="column" gap="2rem">
         <SimpleGrid columns={3} gap="2.5rem" width="100%">
           {sites.map((site) => (
-            <Card key={site.id} width="100%">
-              <CardHeader>
-                <Link href={`/sites/${site.id}`} as={NextLink}>
-                  {site.name}
-                </Link>
-              </CardHeader>
-            </Card>
+            <Link
+              href={`/sites/${site.id}`}
+              as={NextLink}
+              textDecoration="none"
+              color="links.neutral-hover"
+              _hover={{
+                textDecoration: "none",
+                color: "links.neutral-hover",
+              }}
```

Comment:
cc @sehyunidaaa do we also need an `_active/_focus` state transition?

Reply:
no longer needed after using LinkBox